scripts/hid_configurator/configurator_cli.py

```diff
@@ -8,6 +8,13 @@
 import os
 import pprint
 
+if os.name == "nt":
+    try:
+        script_dir = os.path.dirname(os.path.abspath(__file__))
+        os.add_dll_directory(script_dir)
+    except AttributeError:
+        print('Cannot add DLL directory. Consider using Python 3.8 or newer')
```

Comment:
We could require Python >= 3.8 (and state here that `you must use Python 3.8 or newer` - consider is not a right word then; btw. maybe we could also inform about minimal Python version in doc?).

Alternatively we can allow user to still use older Python version, remove this warning and let old Python rely on old ways to find dll files (make sure to update doc then).

Reply:
I thought that we can leave it as I did. If it works for someone he can ignore my print warning. I don't want to support old versions of python but want to give a notice to someone who is using old version that he might want to update it

Reply:
`Cannot add directory containing configurator_cli.py script to DLL search path. Please update to Python 3.8+ or make sure that hidapi.dll can be located by Python version that you use.`